tf2onnx/tf_loader.py

```diff
@@ -684,8 +684,12 @@ def tf_optimize_grappler(input_names, output_names, graph_def, fold_constant=Non
     # depends on so for now don't turn this on, fold_constant is always enabled now.
     rewrite_options.optimizers[:] = [
         # 'pruning', 'constfold', 'arithmetic', 'dependency', 'function',
-        'constfold', 'function'
+        'function', 'dependency'
```

Comment:
Since experimental_disable_folding_quantization_emulation is not available for some versions, we still need to bring "constfold" optimizer back in those cases.